File: w11/d02_warmup/fizzbuzz.rb


(1..100).each do |n|
	fizz = n%3 == 0
	buzz = n%5 == 0
	puts "FizzBuzz" if fizz && buzz
	puts "Fizz" if fizz
	puts "Buzz" if buzz
	puts n if !fizz && !buzz
end

File: w11/d02_warmup/fizzbuzz.py
# numbers = list(range(1, 101))

for n in list(range(1, 101)):
	fizz = n%3 == 0
	buzz = n%5 == 0
	word_fizz = "fizz"
	word_buzz = "buzz"

	if fizz and buzz:
		print word_fizz + word_buzz
	elif fizz:
		print word_fizz
	elif buzz:
		print word_buzz
	else:
		print n
